lib/Lifters/FunctionLifter.cpp

```diff
@@ -1056,43 +1087,47 @@ void FunctionLifter::VisitInstruction(remill::Instruction &inst,
     case remill::Instruction::kCategoryError:
       VisitError(inst, delayed_inst, block);
       break;
-    case remill::Instruction::kCategoryNormal: VisitNormal(inst, block); break;
-    case remill::Instruction::kCategoryNoOp: VisitNoOp(inst, block); break;
+    case remill::Instruction::kCategoryNormal:
```

Comment:
Extract as a standalone function/local lambda and replace `break` with `return`? It won't give that much gains though, but it should save a few line + make this function a bit more readable, which maybe is worth it?